docs_user/modules/proc_adopting-the-identity-service.adoc

```diff
@@ -72,9 +72,16 @@ done
 
 .Verification
 
+. Wait for the `OpenStackControlPlane` resource to become `Ready`:
++
+----
+$ oc wait --for=condition=Ready --timeout=1m OpenStackControlPlane openstack
+----
+
 . Verify that you can access the `OpenStackClient` pod. For more information, see link:{defaultURL}/maintaining_the_red_hat_openstack_services_on_openshift_deployment/assembly_accessing-the-rhoso-cloud#proc_accessing-the-OpenStackClient-pod_cloud-access-admin[Accessing the OpenStackClient pod] in _Maintaining the {rhos_long_noacro} deployment_.
 
-. Confirm that the {identity_service} endpoints are defined and are pointing to the control plane FQDNs:
+. Using the `OpenStackClient` pod remote shell, verify that the {identity_service}
+endpoints are defined and are pointing to the control plane FQDNs:
 +
 ----
 $ openstack endpoint list | grep keystone
```

Comment:
This command uses alias created before. Maybe ` Using the `OpenStackClient` pod remote shell` should be explained above in alias creation step?

Reply:
Ah i didn't scan for the alias, i thought this was missing a "run this command in the client pod" explanation. But since we rely on the alias instead, i'll undo this edit and leave the original formulation as-is.